store/contentSchemas.json

```
@@ -0,0 +1,36 @@
+{
+  "definitions": {
+    "Video": {
+      "properties": {
+        "src": {
+          "title": "admin/editor.video.src.title",
+          "$ref": "app:vtex.native-types#/definitions/url",
+          "default": ""
+        },
+        "width": {
+          "title": "admin/editor.video.width.title",
+          "$ref": "app:vtex.native-types#/definitions/text",
```

This ref points to a type that can be formatted and I think this is not the case here

```suggestion
          "type": "string",
```